### ISSUE TITLE
Decrypt prompt is far to the right on widescreen monitors

### ISSUE THREAD

**Issue by WatchMkr** [pop-os/plymouth-theme#22]
It started sometime in September and may have been introduced with a Plymouth update.

![IMG_0054](https://user-images.githubusercontent.com/3430283/139915776-2f41e4b7-b05d-4a6d-b8d2-32372b15cef9.jpg)

**Comment by leviport** [comment]
Looks like this might be a duplicate of https://github.com/pop-os/plymouth-theme/issues/19

**Comment by 13r0ck** [comment]
Should be closed because of #25

**Comment by leviport** [comment]
Closing because we think it's fixed. Can reopen if we're mistaken.

**Comment by crawfxrd** [comment]
This will happen on oryp6 and kudu6 if the NVIDIA modules are added to initramfs, which is needed for Plymouth to show on external displays.

**Comment by 13r0ck** [comment]
@crawfxrd what are the steps to add the modules? I am trying to reproduce this (and test a fix on the kudu6)

**Comment by crawfxrd** [comment]
```
echo "nvidia-drm" | sudo tee /usr/share/initramfs-tools/modules.d/s76-nvidia-drm.conf
sudo update-initramfs -k all
```

**Comment by 13r0ck** [comment]
Thanks! This is fixed on the kudu with #28
